mappings/net/minecraft/network/packet/s2c/play/TeamS2CPacket.mapping

```diff
@@ -1,5 +1,10 @@
 CLASS net/minecraft/class_5900 net/minecraft/network/packet/s2c/play/TeamS2CPacket
 	FIELD field_29151 packetType I
+		COMMENT <ul><li>0 - Create/Update team with players</li>
+		COMMENT <li>1 - Remove player/team</li>
+		COMMENT <li>2 - Update team</li>
+		COMMENT <li>3 - Change player team</li>
+		COMMENT <li>4 - unused?</li></ul>
```

Comment:
```suggestion
		COMMENT <li>3 - Change player team</li></ul>
```
Never a good idea to name or refer to things that are unused. Especially in snapshots as they can get outdated quickly.

Reply:
Yeah, since this is an int, it can have 5, 6, 7, 8, ... and a lot more values. the case 4 is only worthy of documentation if it differs from those other cases, which are also unused.

Reply:
Hey guess what, 4 is now used since 21w11a :laughing: